strongbox-security/strongbox-user-management/src/main/java/org/carlspring/strongbox/users/domain/AccessModelPathPrivileges.java

```diff
@@ -36,10 +37,10 @@ public AccessModelPathPrivileges(final UserPathPrivilegesDto delegate)
     }
 
 
-    private Set<String> immutePrivileges(final Set<PrivilegeDto> source)
+    private Set<AccessModelPrivelege> immutePrivileges(final Set<PrivilegeDto> source)
```

Comment:
Cosmetic: There's a typo here in the class' name. It should be `AccessModelPrivilege` (not `AccessModelPrivElege`.

Reply:
done